src/libs/ReportUtils.js

```diff
@@ -1295,7 +1321,8 @@ function isWaitingForIOUActionFromCurrentUser(report) {
     }
 
     // Money request waiting for current user to add their credit bank account
-    if (report.hasOutstandingIOU && report.ownerAccountID === currentUserAccountID && report.isWaitingOnBankAccount) {
+    // hasOutstandingIOU will be false if the user paid, but isWaitingOnBankAccount will be true if user don't have a wallet or bank account setup
+    if ((report.hasOutstandingIOU || report.isWaitingOnBankAccount) && report.ownerAccountID === currentUserAccountID) {
```

Comment:
This condition is still wrong. It causes regression above ^

Reply:
I don't think so, I removed it and tested it still shows.

Reply:
I am not seeing GBR after reverting this condition.

Reply:
@situchan I think that dot is intentional, it's not showing in main (tested) because the condition is wrong (the RCA for GBR bug). Maybe you're trying in a new account.

> // Money request waiting for current user to add their credit bank account

This feels like intentional.

Reply:
> I am not seeing GBR after reverting this condition

Means `report.hasOutstandingIOU && report.ownerAccountID === currentUserAccountID` this would be the conditon right?!

Reply:
you haven't fully reverted code.
`report.isWaitingOnBankAccount` is missing

Reply:
@situchan Fixed, Can you please check?